stackslib/src/config/mod.rs

```diff
@@ -2258,52 +2258,435 @@ impl Default for MinerConfig {
+    /// and established HTTP connections.
+    /// It serves as a hard limit to prevent the node from exhausting operating system
+    /// resources related to socket descriptors.
+    ///
+    /// Default: 800.
     pub max_sockets: Option<u64>,
+    /// Minimum interval (in seconds) between the start of consecutive neighbor discovery walks.
+    ///
+    /// The node periodically performs "neighbor walks" to discover new peers and maintain
+    /// an up-to-date view of the P2P network topology. This setting controls how frequently
+    /// these walks can be initiated, preventing excessive network traffic and processing.
+    ///
+    /// Default: 60 seconds.
     pub walk_interval: Option<u64>,
+    /// Probability (0.0 to 1.0) of forcing a neighbor walk to start from a seed/bootstrap peer.
+    ///
+    /// This probability applies only when the node is not in Initial Block Download (IBD)
+    /// and is already connected to at least one seed/bootstrap peer.
+    /// Normally, in this situation, the walk would start from a random inbound or outbound peer.
+    /// However, with this probability, the walk is forced to start from a seed peer instead.
+    /// This helps ensure the node periodically re-establishes its network view from trusted entry points.
+    ///
+    /// Default: 0.1 (10%).
     pub walk_seed_probability: Option<f64>,
+    /// Frequency (in milliseconds) for logging the current P2P neighbor list at the DEBUG level.
+    ///
+    /// If set to a non-zero value, the node will periodically log details about its currently
+    /// established P2P connections (neighbors). Setting this to 0 disables this periodic logging.
+    ///
+    /// Default: 60_000 ms (1 minute).
     pub log_neighbors_freq: Option<u64>,
+    /// Maximum time (in milliseconds) to wait for a DNS query to resolve.
+    ///
+    /// When the node needs to resolve a hostname (e.g., from a peer's advertised `data_url`
+    /// or an Atlas attachment URL) into an IP address, it initiates a DNS lookup.
+    /// This setting defines the maximum duration the node will wait for the DNS server
+    /// to respond before considering the lookup timed out.
+    ///
+    /// Default: 15_000 ms, (15 seconds).
     pub dns_timeout: Option<u64>,
+    /// Maximum number of concurrent Nakamoto block download requests allowed.
+    ///
+    /// This limits how many separate block download processes for Nakamoto tenures
+    /// (both confirmed and unconfirmed) can be active simultaneously. Helps manage
+    /// network bandwidth and processing load during chain synchronization.
+    ///
+    /// Default: 6.
     pub max_inflight_blocks: Option<u64>,
+    /// Maximum number of concurrent Atlas attachment download requests allowed.
+    ///
+    /// This limits how many separate download requests for Atlas data attachments
+    /// can be active simultaneously. Helps manage network resources when fetching
+    /// potentially large attachment data.
+    ///
+    /// Default: 6.
```

Comment:
I'm tempted to say this may be deprecated since atlas is (or will be) deprecated. is this key/val used elsewhere in the code?